### ISSUE TITLE
`xm dev` throws "fs.rmSync is not a function"

### ISSUE THREAD

**Issue by bakkerjoeri** [giuseppeg/xm#13]
When running `xm dev` when there's an `.xm` cache folder present, I get:

```
(...)/xm/index.js:86
  fs.rmSync(outDir, { recursive: true, force: true });
     ^

TypeError: fs.rmSync is not a function
```

Could have been introduced by the change on index.js:86 here https://github.com/giuseppeg/xm/compare/v1.0.5...v1.0.6 

**Steps to reproduce:**
1. Create a basic project, like one from the examples.
2. Run `xm dev`
3. Visit your project once and make sure you now see a `.xm` folder in the root of your project.
4. Stop the `xm dev` process.
5. Run it again

**Comment by giuseppeg** [comment]
This is only supported by new versions of node -.- I'll fix it today

**Comment by giuseppeg** [comment]
fixed in `1.0.7`

**Comment by qubist** [comment]
If anyone's here from searching after having this issue elsewhere, [updating node.js](https://stackoverflow.com/questions/8191459/how-do-i-update-node-js) fixed it for me.
